backend/src/main/java/org/example/config/oauth/OAuth2SuccessHandler.java

```diff
@@ -67,6 +70,14 @@ private void addRefreshTokenToCookie(HttpServletRequest request, HttpServletResp
 		CookieUtil.addCookie(response, REFRESH_TOKEN_COOKIE_NAME, refreshToken, cookieMaxAge);
 	}
 
+	private void addAccessTokenToCookie(HttpServletRequest request, HttpServletResponse response,
```

Comment:
코드가 120줄을 넘길 경우 다음과 같이 개행 맞춰주시면 보기 더 좋을것 같습니다!

```suggestion
	private void addAccessTokenToCookie(
        HttpServletRequest request, 
        HttpServletResponse response,
		String accessToken
    ) {
		int cookieMaxAge = (int)ACCESS_TOKEN_DURATION.toSeconds();

		CookieUtil.deleteCookie(request, response, ACCESS_TOKEN_COOKIE_NAME);
		CookieUtil.addCookie(response, ACCESS_TOKEN_COOKIE_NAME, accessToken, cookieMaxAge);
	}
```